component/src/main/java/com/celements/photo/container/ImageUrlDim.java

```diff
@@ -0,0 +1,35 @@
+package com.celements.photo.container;
+
+public class ImageUrlDim {
+
+  private String url;
+  private String width = "";
+  private String height = "";
+
+  public ImageUrlDim(String url, int width, int height) {
+    this.url = url;
+    if (width > 0) {
```

Comment:
no readable width / height is no problem. Those are just two more tags which are either added or not, so throwing an Exception would not allow to add any images without size, which is neither necessary, nor intended. However I changed it to an Optional.